src/Uno.UWP/UI/Composition/Compositor.Skia.cs

```diff
@@ -59,15 +90,9 @@ private static void RenderVisual(SKSurface surface, SKImageInfo info, Visual vis
 
 				surface.Canvas.SetMatrix(visualMatrix);
 
-				if(visual.Clip is InsetClip insetClip)
-				{
-					surface.Canvas.ClipRect(new SKRect {
-						Top = insetClip.TopInset,
-						Bottom = insetClip.BottomInset,
-						Left = insetClip.LeftInset,
-						Right = insetClip.RightInset
-					});
-				}
+				ApplyClip(surface, visual);
+
+				using var opacityDisposable = PushOpacity(visual.Opacity);
```

Comment:
Codacy found an issue: [Remove the unused local variable 'opacityDisposable'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5924508)